app-misc/task/task-2.5.0.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit eutils cmake-utils bash-completion-r1
+
+DESCRIPTION="Taskwarrior is a command-line todo list manager"
+HOMEPAGE="http://taskwarrior.org/"
+SRC_URI="http://taskwarrior.org/download/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~x86 ~x64-macos"
+IUSE="gnutls vim-syntax zsh-completion"
+
+DEPEND="sys-libs/readline:0
```

Comment:
sys-libs/readline will introduce subslots in the future. These subslots will match SONAME major version (see sys-libs/readline/readline-7.0_beta.ebuild). Thus you most probably want ':0=' here. See link below for the info.